docs/guides/using-beluga-amcl.md

```diff
@@ -1,3 +1,153 @@
+
+    <!-- Load map -->
+    <node pkg="nav2_map_server" exec="map_server" name="map_server" output="screen">
+        <param name="yaml_filename" value="$(var map_path)" />
+    </node>
+
+    <!-- Start AMCL -->
+    <node pkg="beluga" exec="amcl" name="amcl" output="screen">
+        <param name="use_sim_time" value="false" />
+        <param name="min_particles" value="500" />
+        <param name="max_particles" value="2000" />
+        <param name="kld_err" value="0.05" />
+        <param name="odom_alpha1" value="0.2" />
+        <param name="odom_alpha2" value="0.2" />
+        <param name="odom_alpha3" value="0.2" />
+        <param name="odom_alpha4" value="0.2" />
+        <param name="z_hit" value="0.95" />
+        <param name="z_rand" value="0.05" />
+        <param name="laser_likelihood_max_dist" value="2.0" />
+        <param name="laser_model_type" value="likelihood_field" />
+    </node>
+</launch>
+```
+
+## Running Beluga AMCL
+
+Now, it’s time to run Beluga on your robot!
+
+1. **Launch your system** and your custom launch file alongside with it:
+
+     ```bash
+     ros2 launch beluga.launch.xml
+     ```
+
+2. **Verify your robot is properly localized** using `rviz2`:
+
+     ```bash
+     ros2 run rviz2 rviz2
+     ```
+
+    Use `map` as your global fixed frame. Use the `Map` and `LaserScan` displays to assess your robot localization. If scans and map do not line up, use the `2D Pose Estimate` tool to reset your robot pose. You can also use `PoseArray` and `MarkerArray` displays to visualize the particle cloud. When moving, it should quickly converge near the robot's true pose.
+
+3. **Tweak Beluga AMCL as necessary** if localization isn’t as accurate as expected. For example, you can:
+
+   - **Adjust the number of particles**. If localization is unstable (e.g., the robot’s position jumps), consider increasing the minimum number of particles (`min_particles`) to improve the pose estimate. If localization is slow (especially in large environments), consider lowering the maximum number of particles (`max_particles`) to reduce the computational load.
+   - **Tune the odometry noise model**. If your odometry source is noisy and localization drifts, consider updating `odom_alpha1` thru `odom_alpha5` to better reflect that noise.
+   - **Tune the sensor model**. If you LiDAR is noisy, consider increasing `z_rand` to account for bad range measurements.
+
+## Troubleshooting
+
+Some common issues you may find along the way include:
+
+1. **Beluga AMCL fails to converge**
+
+   - **Symptom:** the particle cloud doesn't cluster around the robot's true pose.
+   - **Possible causes:**
+     - Incorrect LiDAR or odometry data.
+     - Map doesn't match the actual environment.
+   - **Crosschecks:**
+     - Verify the `/scan` topic data is correct using `ros2 topic echo /scan`.
+     - Verify the `odom` to `base_link` transforms are correct using `ros2 run tf2_ros tf2_echo odom base_link`.
```

Comment:
Also check lidar to base_link.

Reply:
True that. Done in https://github.com/Ekumen-OS/beluga/pull/447.